README.md

```diff
@@ -1,128 +1,82 @@
 
-Here is the list of the technologies that we can use in the test:
+## Getting Started
 
-- [**Terraform**](https://www.terraform.io/)
-- [**Ansible**](https://www.ansible.com/)
-- [**Jekyll**](https://jekyllrb.com/) or [**Hugo**](https://gohugo.io/) static site generator
-- **Git**
-- Programming Language: **Python** / **Go** / **Shell** or any other language.
+Notes: Since DigtalOcean's network will be disturbed by [GFW](https://en.wikipedia.org/wiki/Great_Firewall), maybe you need to prepare a VPN first.
 
-## Task
+### Prepare
 
-The test is composed of 2 components:
-- a **dev** related task; where you'll be expected to prepare a simple site using a static site generator,
-- an **ops** related task; where you'll be expected to spawn a server to run and operate the site you've created.
+#### Set digtalocean token
 
-**Make sure you create appropriate documentation on how to run your code and playbook. Create a `README.md` file for that purpose, or store the documentation in a `docs/` folder.**
+[How to Create a Personal Access Token](https://www.digitalocean.com/docs/apis-clis/api/create-personal-access-token/)
 
-### Development task
+```sh
+export DO_TOKEN=bbxxxxxx
+```
 
-You will need to create:
-- a site, based on a static site generator (Jekyll or Hugo),
-- a script that will create a new post in your site, and update some meta data information (version - see below), eventually pushing the changes back to github.
+#### Create ssh key and add them into DigtalOcean
 
-A few suggestions / recommendations:
+```sh
+# the key files will be saved as ./id_rsa and ./id_rsa.pub
+make create-ssh-key
+```
 
-- Choose whichever static site generator to use and prepare the basic site, plenty of tutorial are available online:
-    - [Jekyll](https://jekyllrb.com)
-    - [Hugo](https://gohugo.io)
-- Prepare templates for your site generator:
-    - base template for your site to display the `version` in the footer of the pages, and a list of the posts on the landing page,
-    - post template that will display the content of individual pages
-- Prepare a script (choose whichever language) that will take one parameter:
-    - either `dev`, in which case, it will:
-        - create a new post using markdown with yaml frontmatter format,
-        - use the output of [`fortune`](http://manpages.ubuntu.com/manpages/xenial/man6/fortune.6.html) command as content,
-        - and increment the version of the site by `0.0.1`,
-        - compile the site,
-        - commit and push the sources (markdown files) and build site (html) back to github
-    - or `staging`, in which case, it will:
-        - take the last known version of the site,
-        - increment the version of the site by `0.1.x` (e.g. move from `0.1.5` to `0.2.0`)
-        - compile the site
-        - commit, tag and pus to github
+[How-to Add SSH Keys to DigtalOcean](https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/)
```

Comment:
Why not use https://registry.terraform.io/providers/digitalocean/digitalocean/latest/docs/resources/ssh_key ?

Reply:
thanks for your mention, it's helpful.